cmd/ddev/cmd/root.go

```diff
@@ -212,9 +212,9 @@ func checkDdevVersionAndOptInInstrumentation(skipConfirmation bool) error {
 			output.UserOut.Print("Congratulations, you seem to have a new DDEV version.")
 			home, _ := homedir.Dir()
 			if globalconfig.DdevGlobalConfig.MutagenEnabledGlobal && fileutil.IsDirectory(filepath.Join(home, ".mutagen")) {
-				output.UserOut.Print("Please note that the global ~/.mutagen folder is no longer used by DDEV so you can delete it if it's no longer used by other applications.")
+				output.UserOut.Print("Please note that the global ~/.mutagen folder is no longer used by DDEV. Delete the folder if it is no longer used by any other application.")
```

Comment:
I apologize, but in use I've found this to be really annoying, and think we should remove it.